DigitalLearningSolutions.Data/Services/CentreRegistrationPromptsService.cs

```diff
@@ -260,6 +261,101 @@ public string GetCentreRegistrationPromptNameAndNumber(int centreId, int promptN
+                1,
+                result.CentreRegistrationPrompt1Id,
+                result.CentreRegistrationPrompt1Prompt,
+                result.CentreRegistrationPrompt1Options,
+                result.CentreRegistrationPrompt1Mandatory,
+                delegateAccount.Answer1
+            );
+            if (prompt1 != null)
+            {
+                list.Add(prompt1);
+            }
+
+            var prompt2 = PromptHelper.PopulateCentreRegistrationPromptWithAnswer(
+                2,
+                result.CentreRegistrationPrompt2Id,
+                result.CentreRegistrationPrompt2Prompt,
+                result.CentreRegistrationPrompt2Options,
+                result.CentreRegistrationPrompt2Mandatory,
+                delegateAccount.Answer2
+            );
+            if (prompt2 != null)
+            {
+                list.Add(prompt2);
+            }
+
+            var prompt3 = PromptHelper.PopulateCentreRegistrationPromptWithAnswer(
+                3,
+                result.CentreRegistrationPrompt3Id,
+                result.CentreRegistrationPrompt3Prompt,
+                result.CentreRegistrationPrompt3Options,
+                result.CentreRegistrationPrompt3Mandatory,
+                delegateAccount.Answer3
+            );
+            if (prompt3 != null)
+            {
+                list.Add(prompt3);
+            }
+
+            var prompt4 = PromptHelper.PopulateCentreRegistrationPromptWithAnswer(
+                4,
+                result.CentreRegistrationPrompt4Id,
+                result.CentreRegistrationPrompt4Prompt,
+                result.CentreRegistrationPrompt4Options,
+                result.CentreRegistrationPrompt4Mandatory,
+                delegateAccount.Answer4
+            );
+            if (prompt4 != null)
+            {
+                list.Add(prompt4);
+            }
+
+            var prompt5 = PromptHelper.PopulateCentreRegistrationPromptWithAnswer(
+                5,
+                result.CentreRegistrationPrompt5Id,
+                result.CentreRegistrationPrompt5Prompt,
+                result.CentreRegistrationPrompt5Options,
+                result.CentreRegistrationPrompt5Mandatory,
+                delegateAccount.Answer5
+            );
+            if (prompt5 != null)
+            {
+                list.Add(prompt5);
+            }
+
+            var prompt6 = PromptHelper.PopulateCentreRegistrationPromptWithAnswer(
+                6,
+                result.CentreRegistrationPrompt6Id,
+                result.CentreRegistrationPrompt6Prompt,
+                result.CentreRegistrationPrompt6Options,
+                result.CentreRegistrationPrompt6Mandatory,
+                delegateAccount.Answer6
+            );
+            if (prompt6 != null)
+            {
+                list.Add(prompt6);
+            }
+
+            return list;
+        }
+
+        [Obsolete]

```

Comment:
Do we expect people to need to use it in the future, or should they always be using new version? I expect it's the latter, and if so it is appropriate to mark this obsolete.
It would be good to have a message in the attribute explaining what should be used instead.